scripts/core/exp-launch-attributes.yml

```diff
@@ -0,0 +1,131 @@
+type: header
+desc: "Intel $OneApi Unified Runtime Experimental APIs for (kernel) Launch Attributes"
+ordinal: "99"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "The extension string that defines support for the Launch Attributes extension, which is returned when querying device extensions."
+name: $X_LAUNCH_ATTRIBUTES_EXTENSION_STRING_EXP
+value: "\"$x_exp_launch_attributes\""
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Specifies a launch attribute id"
+name: $x_exp_launch_attribute_id_t
+analogue:
+    - "**CUlaunchAttributeID**"
+etors:
+    - name: IGNORE
+      desc: "the attribute has no effect"
+    - name: COOPERATIVE
+      desc: "whether to launch a cooperative kernel"
+    - name: CLUSTER_DIMENSION
+      desc: "cluster dimensions"
+--- #--------------------------------------------------------------------------
+type: union
+desc: "Specifies a launch attribute value"
+name: $x_exp_launch_attribute_value_t
+tag: $x_exp_launch_attribute_id_t
+analogue:
+    - "**CUlaunchAttributeValue**"
+members:
+    - type: uint32_t[3]
+      name: clusterDim
+      desc: "[in] cluster dimensions (x, y, z)"
+      tag: $X_EXP_LAUNCH_ATTRIBUTE_ID_CLUSTER_DIMENSION
+    - type: int
+      name: cooperative
+      desc: "[in] non-zero value indicates a cooperative kernel"
+      tag: $X_EXP_LAUNCH_ATTRIBUTE_ID_COOPERATIVE
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Kernel launch attribute"
+name: $x_exp_launch_attribute_t
+analogue:
+    - "**cuLaunchAttribute**"
+members:
+    - type: $x_exp_launch_attribute_id_t
+      name: id
+      desc: "[in] Launch attribute id"
+      init: $X_EXP_LAUNCH_ATTRIBUTE_ID_IGNORE
+    - type: $x_exp_launch_attribute_value_t
+      name: value
+      desc: "[in][tagged_by(id)] Launch attribute value"
+      init: nullptr
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Launch kernel with custom Launch attributes"
+class: $xEnqueue
+name: KernelLaunchCustomExp
+ordinal: "0"
+analogue:
+    - "**cuLaunchKernelEx**"
+details:
+    - "Launches the kernel using the specified launch attributes"
+    - "If numAttrsInLaunchAttrList == 0 then a regular kernel launch is used: `urEnqueueKernelLaunch`"
+    - "Consult the appropriate adapter driver documentation for details of adapter specific behavior and native error codes that may be returned."
+params:
+    - type: $x_queue_handle_t
+      name: hQueue
+      desc: "[in] handle of the queue object"
+    - type: $x_kernel_handle_t
+      name: hKernel
+      desc: "[in] handle of the kernel object"
+    - type: uint32_t
+      name: workDim
+      desc: "[in] number of dimensions, from 1 to 3, to specify the global and work-group work-items"
+    - type: const size_t*
+      name: pGlobalWorkSize
```

Comment:
I see you have opted not to have a global offset param. Can you confirm that this was intentional?

Reply:
Yup, @npmiller gave me the green light.

Reply:
OK nice one